tests/testThreads2.py

```diff
@@ -23,6 +23,7 @@
 
 import unittest
 
+from past.builtins import long
```

Comment:
With pybind11 a C++ function with return type `int` gives a `long` on Python 2. Not sure if this is a bug or design. I have filed a ticket upstream, but in the mean time this is needed.